text/tls-and-alpn.rst

```diff
@@ -0,0 +1,380 @@
+TLS and ALPN
+============
+
+This RFC proposes to change the transport of EdgeDB frontend connections to TLS,
+and use ALPN for multiplexing protocol selection.
+
+
+Motivation
```

Comment:
BTW, we have UNIX sockets (and AFAIK we'll continue to have them) -- IMO we should reserve UNIX sockets for clear-text binary proto. cc @elprans

Reply:
For the record, in a video discussion we agreed to drop the UNIX sockets for non-admin purposes, and keep the admin UNIX socket with clear-text binary protocol only. Meanwhile, we'll drop the (hopefully already deprecated) admin connection option in all bindings, and the CLI will be the only client that connects to the admin UNIX socket.